panic.go

```diff
@@ -6,7 +6,7 @@ import "testing"
 // Does not detect panics in separate goroutines.
 func Panics(t *testing.T, functionToTest func()) {
 	defer func() {
-		// This deferred function will recover it if it panics
+		// This deferred function will capture the panic if it panics
```

Comment:
OK, "capture the panic if it panics" sounds weird, too, but better than before. I suppose you could clarify with "capture a panic from the test function" or some such, but you didn't hear it from me... I'm just approving and moving on. 😬

Reply:
I created an even better message:
> This deferred function will capture the panic (if present) from the function under test.

I'll push that change.

Reply:
Even much more betterer!

(Technically it also fails if the test function *doesn't* panic, but it's a simple enough func and you've captured the essential purpose.)